README.md

```diff
@@ -108,11 +108,11 @@ If your Strapi instance uses API tokens, configure the SDK like this:
 
 ```typescript
 const sdk = strapi({
+  // Endpoint configuration
   baseURL: 'http://localhost:1337/api',
-  auth: {
-    strategy: 'api-token',
-    options: { token: 'your-api-token-here' },
-  },
+  // Auth configuration
+  authStrategy: 'api-token',
```

Comment:
`authStrategy` or `auth_strategy`? :thinking:

Reply:
do we need authStrategy at all if we're doing this? We could just have an `api-token` property and use api-token strategy if that exists, and then if we add another method later like u&p we could allow a `user: { email, password }` to use instead

Reply:
or if we want it really minimalistic, `auth` could be either string or object, and string could be api token and object could be for all other cases, but I think then it is less clear

Reply:
I've modified the behavior, it now asks for `auth?: string` as the API token and doesn't require to pass an `authStrategy`